examples/vumps/vumps_hubbard_extended.jl

```diff
@@ -120,19 +121,24 @@ Sz_finite = ITensors.expect(ψfinite, "Sz")[nfinite:(nfinite + 1)]
 
 energy_exact = reference(model, Observable("energy"); U=model_params.U / model_params.t)
 
+corr_infinite = ITensors.correlation_matrix(finite_mps(ψ, 1:10), "Nup", "Ndn")
```

Comment:
Should be able to change `ITensors.correlation_matrix` to `correlation_matrix` since it is exported from `ITensors.jl`.

Reply:
Also I think you need to add the keyword argument `sites=2:11` to skip the first "site" which has a link index.